pkg/config/config.go

```diff
@@ -43,6 +43,10 @@ type Config struct {
 	// If ShowLocationExpr is true whatis will print the DWARF location
 	// expression for its argument.
 	ShowLocationExpr bool `yaml:"show-location-expr"`
+	
+	// Source list line-number preference: dark (default, blue) or
+	// light (bright yellow)
+	SrcListLineColor bool `yaml:"source-list-line-bright"`
```

Comment:
At least make the name of the field match the yaml name: `SrcListLineColor` → `SrcListLineBright`.

Reply:
OK! Thanks for the feedback. I will address all those & resubmit ASAP.